jobs/api/v4/JobsSample/CreateClientEventSample.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Google.Cloud.Talent.V4Beta1;
+using Google.Protobuf.WellKnownTypes;
+using static Google.Cloud.Talent.V4Beta1.JobEvent.Types;
+
+namespace GoogleCloudSamples
+{
+    internal class CreateClientEventSample
+    {
+        // [START job_search_create_client_event]
+        public static object CreateClientEvent(string projectId,
+            string tenantId, string requestId, string eventId)
+        {
+            EventServiceClient eventServiceClient = EventServiceClient.Create();
+
+            TenantName parent = new TenantName(projectId, tenantId);
+
+            long seconds = 1L;
+            Timestamp createTime = new Timestamp
+            {
+                Seconds = seconds
+            };
+
+            // The type of event attributed to the behavior of the end user.
+            JobEventType type = JobEventType.View;
+
+
+            // List of job names associated with this event.
+            string jobsElement = "projects/[Project ID]/tenants/[Tenant ID]/jobs/[Job ID]";
```

Comment:
Is there something like JobName that you can use to parse these?
And also, shouldn't you be using the values passed as parameters?

Reply:
There could be multiple jobs more than two. I thought taking all of jobs from the params was not good idea. How we should do this?